ci-operator/step-registry/hypershift/kubevirt/run-e2e-external/hypershift-kubevirt-run-e2e-external-chain.yaml

```diff
@@ -43,8 +57,10 @@ chain:
       - env: CI_HYPERSHIFT_OPERATOR
         name: pipeline:hypershift-operator
     env:
-      - default: ${SHARED_DIR}/management_cluster_kubeconfig
+      - default: ${SHARED_DIR}/nested_kubeconfig
         name: KUBECONFIG
```

Comment:
The test assets still don't seem to indicate that the mgmt cluster is the kubevirt cluster. I'm seeing the aws nodes dumped for this job. https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/40447/rehearse-40447-pull-ci-openshift-hypershift-main-e2e-kubevirt-aws-ovn/1673634711257223168/artifacts/e2e-kubevirt-aws-ovn/run-e2e-external/artifacts/TestCreateCluster_PreTeardownClusterDump/cluster-scoped-resources/core/nodes/

I wonder if this default isn't being honored because the `KUBECONFIG` env var already exists when this test step executes.

Reply:
i added some debug commands to check on which cluster we are working with, in several locations in the workflow.